galaxy/templates/deployment-job.yaml

```diff
@@ -35,9 +35,11 @@ spec:
           command: ["python", "/galaxy/server/scripts/galaxy-main"]
           args: ["-c", "/galaxy/server/config/galaxy.yml", "--server-name", "$(POD_NAME)", "--attach-to-pool", "job-handlers"]
           volumeMounts:
-            - name: galaxy-job-conf-file
-              mountPath: /galaxy/server/config/job_conf.xml
-              subPath: job_conf.xml
+            {{- range $num, $entry := .Values.galaxyConfig }}
+            - name: galaxy-conf-files
+              mountPath: {{ $entry.mountPath }}
+              subPath: {{ $entry.name }}
```

Comment:
Not a big deal, but using subPath causes the files to not be visible in the hostPath, due to issue: https://github.com/kubernetes/kubernetes/issues/72991
Of course, we probably won't be using hostPaths other than for dev environments, but just flagging it here.

Reply:
Isn't that bug just in `rkt` environments though? If so, are we ever going to run in that environment?

Reply:
I may have got the wrong link for the bug, but this was the reason we couldn't see the postgres files in the hostPath earlier. The newer version of the postgres chart doesn't use subPath anymore, so it's no longer an issue.